### ISSUE TITLE
Reports on TestSuiteCollection when called from command line

### ISSUE THREAD

**Issue by jchatronLetsignit** [katalon-studio/katalon-studio#19]
Hello everyone, so i have a testSuiteCollection like this : 
TestSuiteCollection (Sequential) : 
 - TestSuite 1
 - TestSuite 2 
 - TestSuite 3

When i launch the TestSuiteCollection with a call from a command line for example :

katalon -noSplash -runMode=console -reportFolder="/reports" -projectPath="PATH" -retry=0 -testSuiteCollectionPath="TestSuiteCollection"

in the report folder katalon overwrite its own report folder. 
When the first test suite (TestSuite 1) start 1 it create the folder in the /reports (for example : 20181113_083747) and write the outputs in it nicely (report.html etc).

But when the second test suite (TestSuite 2) starts it put the reports in the same folder so when the Collection end i have only the report.html of the last Testsuite (TestSuite 3).

PS : when i run this same test suite collection with the katalon tool (without calling it from command line) it run perfectly creating a different folder with the name of the TestSuite like this : 
/reports/{TestSuiteName}/{TimeStamp}/report.html 

It's not the case when calling the Collection with the command line

PS 2 : I have the feeling that with Parallel TestSuiteCollection instead of Sequential Katalon create one folder per TestSuite cheating on the timestamp so it work nicely. But like in my case with a Sequential one each step (TestSuite) is redirected in the main report folder created with the first timestamp therefore each Report.html overwrite the previous one.

Running infos : 
Debian 9.5
Katalon_Studio_Linux_64-5.8.6

**Comment by StellarNear** [comment]
- The command : 
```
katalon -noSplash  -runMode=console -noExit -projectPath="/home/testadmin/localTesting/dars/application.prj" -retry=0 -statusDelay=15 -reportFolder="/home/testadmin/localTesting/launching/tempReports" -testSuiteCollectionPath="Test Suites/Plateform/Tests admin rest"
```

- The run start : 
```
Project 'application' opened
--------------------------------------------------------------------------------
Test Suites/Plateform/Tests admin rest - 20181127_095326................0/38(0%)
--------------------------------------------------------------------------------
Test Suites/Plateform/InitRest - local_chrome - 20181127_095326..........0/3(0%)
Test Suites/Plateform/Scenario 1521 (T.A.C) Sig,Cam,Des - local_chrome - 2018112
7_095327.................................................................0/8(0%)
Test Suites/Plateform/Scenario 1572 (-.A.C) Tea - local_chrome - 20181127_095327
.........................................................................0/4(0%)
Test Suites/Plateform/Scenario 1570 (T.A.C) Sig,Des - local_chrome - 20181127_09
5327.....................................................................0/6(0%)
Test Suites/Plateform/Scenario 1592 (T.A.C) Sig,Des - local_chrome - 20181127_09
5327.....................................................................0/4(0%)
Test Suites/Plateform/Scenario 1568 (T.A.C) Cam,Des - local_chrome - 20181127_09
5327.....................................................................0/5(0%)
Test Suites/Plateform/Scenario 1519 (T.A.L) Cam,Des - local_chrome - 20181127_09
5327.....................................................................0/4(0%)
Test Suites/Plateform/Scenario 1573 (S.A.C) Sub - local_chrome - 20181127_095327
.........................................................................0/4(0%)
--------------------------------------------------------------------------------
```
As you can see the timestamp went to 20181127_095326 to 20181127_095327 during the startup. Only two folders are created. But the Test Suite Collection contains 8 Test Suites. 


- During the run : 
```
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095326...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...
```
Each copy into the same folder (20181127_095327) overwrite the previous report...


- At the end of the run :
```
--------------------------------------------------------------------------------
Test Suites/Plateform/Tests admin rest - 20181127_095326.............38/38(100%)
--------------------------------------------------------------------------------
Test Suites/Plateform/InitRest - local_chrome - 20181127_095326........3/3(100%)
Test Suites/Plateform/Scenario 1573 (S.A.C) Sub - local_chrome - 20181127_095327
.......................................................................4/4(100%)
Test Suites/Plateform/Scenario 1519 (T.A.L) Cam,Des - local_chrome - 20181127_09
5327...................................................................4/4(100%)
Test Suites/Plateform/Scenario 1568 (T.A.C) Cam,Des - local_chrome - 20181127_09
5327...................................................................5/5(100%)
Test Suites/Plateform/Scenario 1592 (T.A.C) Sig,Des - local_chrome - 20181127_09
5327...................................................................4/4(100%)
Test Suites/Plateform/Scenario 1570 (T.A.C) Sig,Des - local_chrome - 20181127_09
5327...................................................................6/6(100%)
Test Suites/Plateform/Scenario 1572 (-.A.C) Tea - local_chrome - 20181127_095327
.......................................................................4/4(100%)
Test Suites/Plateform/Scenario 1521 (T.A.C) Sig,Cam,Des - local_chrome - 2018112
7_095327...............................................................8/8(100%)
--------------------------------------------------------------------------------
Copying report to folder /home/testadmin/localTesting/launching/tempReports/20181127_095327...

testadmin@katalon:~/localTesting/launching$ ls tempReports/
20181127_095326  20181127_095327
```

In each folder only one report.html remain so i have only two report among my eight Test Suites

**Comment by antho49** [comment]
similar issue with docker image...
https://github.com/katalon-studio/docker-images/issues/26

**Comment by jchatronLetsignit** [comment]
Still no update on this ? :(

**Comment by idursto** [comment]
Having the same issue...hoping for an update soon as I would prefer not to have to abandon running the tests in parallel in the CLI...:(

**Comment by jchatronLetsignit** [comment]
Still no news ? :(

**Comment by idursto** [comment]
I just saw 6.0 was released and still no comment on this issue?  Has anyone tested if it is fixed in v6.0 of katalon?  I didnt see any mention of this issue in the release notes...I really don't understand why there is no response to this yet.  When you break it down the age old issue of using a timestamp to name things processing in parallel is not uncommon and normally a very easy/quick thing to address...

**Comment by jchatronLetsignit** [comment]
Are you sure about 6.0 ? https://github.com/katalon-studio/katalon-studio/releases 
I will wait for 6.1 to get a stable release :D

**Comment by jeroengijsman** [comment]
Found another ticket about this issue and i created one in the docker repo but this clearly looks like a katalon issue https://github.com/katalon-studio/docker-images/issues/46

**Comment by jchatronLetsignit** [comment]
Nice thanks for your work !

**Comment by aditki** [comment]
@devalex88 , Now after fixing this, is there a way to generate a single html report for a TestSuiteCollection?

**Comment by devalex88** [comment]
@aditki This feature has been planned and will be available soon. I'll close this issue - please watch our forum for future releases.

**Comment by jchatronLetsignit** [comment]
Is this fix already released ? the katalon version is 6.1.5 but the bug seems still there
